### DISCUSSION TITLE
C:\Users\k\AppData\Local\vcpkg, What does it do？

### DISCUSSION THREAD

**Discussion by KangLin** [microsoft/vcpkg#41670] [Q&A]
My Vpckg is installed at D:\source\vcpkg .
Buf I am find C:\Users\k\AppData\Local\vcpkg . it is takes up a lot of space. What does it do.? Can delete or remove other position?

**Comment by Osyotr** [comment]
It's used for binary caching and to store git trees from registries.
It can be safely deleted if you don't care about binary caching or don't mind to wait for registries to be re-fetched.

**Comment by KangLin** [comment]
Can move it to other folder(driv)?

**Reply by Osyotr** [reply]
You can move binary cache, but I'm not sure if you can move registries cache.
https://learn.microsoft.com/en-us/vcpkg/consume/binary-caching-default
